### ISSUE TITLE
Infinite Nudge: "dead zone" in middle of screen where mouse interactions are utterly broken

### ISSUE THREAD

**Issue by demux4555** [mklierman/SatisfactoryMods#15]
After a few minutes of playing, this mod seems to create an area in the middle of the screen where mouse interactions are all wonky.

This area a.k.a "dead zone" seems to correspond with the "Currently Building" popup thing... https://i.imgur.com/x22JHCM.png

* In the Map, Inventory, Machine screens, etc, the "dead zone" does not react to mouse clicks. At all. 
* But if you happen to drag & drop any items **into** an inventory slot that is in the "dead zone", it is simply **thrown on the ground where you're standing**. 

There is no indicator showing that an Inventory slot is activated with mouseover (background colour changes), and the "dead zone" is simply treated as if your mouse pointer is **outside** the inventory window. 

This is not a fun bug. It took a long time until I noticed that I kept dropping important stuff randomly on the ground all over the map.

I've disabled all other mods. When running this mod, this issue starts. So I am confident there is something in this mod that causes this problem.

**Comment by mklierman** [comment]
This was fixed a few versions ago. Are you sure the mod is up to date?

**Comment by demux4555** [comment]
Yes, the mod was the current version at time of writing.  I started playing
the game in 1.0, so no "old" mods existed on my system. I only started
looking at mods a few days after mod support was restored in 1.0.
